Sources/Tendermint/RESTClient/RESTResponses.swift

```diff
@@ -90,6 +90,11 @@ public struct CheckTransactionResponse: Codable {
 //    type ResultCheckTx struct {
 //        abci.ResponseCheckTx
 //    }
+    let code: UInt32
+    let data: Data // bytes.HexBytes
+    let log: String
+    let codespace: String // TODO: is this necessary?
+    let hash: Data //bytes.HexBytes
 }
```

Comment:
abci has the following type for ResponseCheckTx so it seems there's some properties missing still?

```
   public let code: UInt32
    /// Result bytes, if any.
    public let data: Data
    /// The output of the application's logger. May be non-deterministic.
    public let log: String
    /// Additional information. May be non-deterministic.
    public let info: String
    /// Amount of gas required for the transaction.
    public let gasWanted: Int64
    /// Amount of gas consumed by the transaction.
    public let gasUsed: Int64
    /// Type and Key-Value events for indexing transactions (eg. by account).
    public let events: [Event]
    /// Namespace for the `code`.
    public let codespace: String
```

Reply:
mainly `info`, `gasWanted`, `gasUsed` and `events`
and there's no hash on the responsechecktx type. So not sure where this is coming from?